back/engines/commercial/analytics/db/views/analytics_fact_project_statuses_v01.sql

```diff
@@ -0,0 +1,37 @@
+WITH regular_project_statuses AS -- project statuses that do not need to be calculated
+         (SELECT item_id as project_id, action as status, acted_at as timestamp
+          FROM activities
+          WHERE item_type = 'Project'
+            AND action IN ('created', 'draft', 'published', 'archived', 'deleted')),
+
+     last_statuses_for_continuous_projects AS
+         (SELECT DISTINCT ON (project_id) *
+          FROM regular_project_statuses JOIN projects ON project_id = projects.id
```

Comment:
point of style really, but I would always favour using aliases when joining tables